js/jquery-ui-popup.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * This script is responsible for making the jQuery UI popups take full height
+ * of the screen in fullscreen mode.
+ */
+
+(function ($) {
+  $(document).on('dialogopen', function(e) {
+    var $el = $(e.target);
+
+    if ($el.parent().hasClass('crm-container') && $el.dialog('option', 'resizable')) {
+      $('.crm-dialog-titlebar-resize', $el.parent()).click(function(e) {
```

Comment:
I see multiple usages of `$el.parent()` in the code. Should we create a variable for this to stop unnecessary DOM queries?